app/views/crowdblog/posts/show.html.slim

```diff
@@ -18,19 +18,24 @@
     = previous_link_mobile(@post)
 .posts
   = render :partial => 'crowdblog/posts/post', :object => @post, :locals => { index_view: false }
-  #disqus_thread
 
-  - if Rails.env.production?
-    coffee:
-      disqus_shortname = 'crowdinttech'
+  #disqus_thread
 
-      dsq = document.createElement('script')
-      dsq.type = 'text/javascript'
-      dsq.async = true
-      dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js'
-      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq)
+  javascript:
```

Comment:
@zalapeach I know that you don't put the initial javascript code in this view but what about if you move this javascript logic into a partial, don't you think is better?